stories/Components/UIcomponents/Buttons/Switch/Switch.jsx

```diff
@@ -0,0 +1,64 @@
+import React, { useEffect, useState } from 'react';
+import './switch.scss';
+import '../../../../Atom/Icons/icons.scss';
+
+export function Switch({
+    size = 'default',
+    state = 'default',
+    isToggled = true,
+    showIcon = false,
+    showLabel = false,
+    showValue = false,
+    onToggle,
+    label = 'Label:',
+    valueOn = 'On',
+    valueOff = 'Off',
+    ...args
+  }) {
+    const [toggled, setToggled] = useState(isToggled);
+  
+    useEffect(() => {
+        setToggled(isToggled);
+      }, [isToggled]);
+
+    const handleToggle = () => {
+      if (state !== 'disabled') {
+        setToggled(!toggled);
+        if (onToggle) onToggle(!toggled);
+      }
+    };
+ 
+    const switchClass = `
+      switch
+      switch--${size}
+      switch--${state}
+      ${toggled ? 'switch--toggled' : 'switch--untoggled'}
+    `;
+
+    useEffect(() => {
+      console.log('toggled', toggled);
```

Comment:
@CalebLuster , please remove console.log call, it shouldn't be a part of production code